index.js

```diff
@@ -196,6 +196,10 @@ var BrowserStackBrowser = function (
       video: bsConfig.video !== undefined ? bsConfig.video : true
     }
 
+    if (bsConfig.extraCapabilities) {
```

Comment:
probably should check that it is actually an object, I am also a bit worried about it overriding other args accidentally if they have the same name as another config option. Is this the only way to supply these options?

Reply:
Good point. I can compare the supplied keys with existing configuration keys and either throw and error or quietly ignore them.

Thanks for taking the time to look at this. Very much appreciated :)